known-packages.toml

```diff
@@ -71,6 +71,20 @@ badges = [
   "<a href='https://anaconda.org/conda-forge/yaqd-core'><img src='https://img.shields.io/conda/vn/conda-forge/yaqd-core'></a>",
 ]
 
+[daemons.yaqd-core-python]
+badges = [
+  "<a href='https://gitlab.com/yaq/yaq-python/builds'><img src='https://img.shields.io/gitlab/pipeline-status/yaq/yaq-python'></a>",
```

Comment:
Is this section intentional? 

Should probably use the updated badge even if so... in fact a lot of these probably need updating...

Reply:
copy paste error